apps/functions-subscription/src/adapters/azure/functions/__tests__/data.ts

```diff
@@ -0,0 +1,23 @@
+import { HttpRequest } from '@azure/functions';
+import { CreateSubscription } from '../../../../generated/definitions/internal/CreateSubscription';
+import { UserId } from '../../../../generated/definitions/internal/UserId';
+
+const aUserId = 'aUserId' as UserId;
+
+const aCreateSubscription: CreateSubscription = {
+  userId: aUserId,
+};
+
+/**
+ * This is a function because, if it were an object, the `body` property
+ * would be removed during the execution of the tests.
+ */
```

Comment:
**suggestion (non-blocking)**: I think this comment is not needed in TSDoc.

```suggestion
// This is a function because, if it were an object, the `body` property
// would be removed during the execution of the tests.
```

Reply:
👍